src/terrain/routes/metadata.clj

```diff
@@ -331,7 +331,10 @@
      (service/success-response (apps/list-job-steps analysis-id)))
 
    (POST "/analyses/:analysis-id/stop" [analysis-id :as {:keys [params]}]
-     (service/success-response (apps/stop-job analysis-id params)))))
+     (service/success-response (apps/stop-job analysis-id params)))
+
+   (GET "/analyses/:analysis-id/history" [analysis-id]
```

Comment:
I might have put this endpoint above `GET /analyses/:analysis-id/parameters`, just to keep the endpoints in alphabetical order, but that's also something that can be done later when this namespace is swaggerized.

Reply:
Ah, that seems like a good idea. I'll update it.